app/javascript/packages/verify-flow/services/api.ts

```diff
@@ -1,5 +1,5 @@
 export interface ErrorResponse<Field extends string = string> {
```

Comment:
> this is only for the new flow, right? doesn't affect stuff live in prod (I ask, because we would need to make it forward/back compatible for deploys)

Yes, correct, it's not live yet in production.